assets/base/i18n/en.json

```diff
@@ -2707,7 +2707,7 @@
   "password_send.description": "To reset your password, enter the email address you used to sign up",
   "password_send.email": "Email",
   "password_send.error": "Please enter a valid email address.",
-  "password_send.link": "If the account exists, a password reset email will be sent to: <br/><b>{email}</b><br/><br/>",
+  "password_send.link": "If the account exists, a password reset email will be sent to:",
```

Comment:
the html tags cause issues with rendering formatted text. So I had to remove it. Let me know someone has a better solution for this.

Reply:
I think you need to leave it as is and just create another one specifically for mobile RN (starting with `mobile.`).  Also, you might want to be the first to implement `FormattedMarkdownText`.

Reply:
tried `FormattedMarkdownText` this but it keeps throwing weird errors. so added a new translation key for mobile.

Reply:
error is `invariant-violation-view-config-not-found-for-name-span` which did not help all that much. I will dig into to more and see if i can fix it.

Reply:
@hmhealey I added a new key for mobile and reverted web key as before so I guess this should be good.

Reply:
Hey @enahum I raised a [PR](https://github.com/mattermost/mattermost-webapp/pull/1419) to change the key in webapp. Once it is merged should I update it here as well?